### DISCUSSION TITLE
Can't add a new config_context in Netbox

### DISCUSSION THREAD

**Discussion by gabrieldslacerda** [netbox-community/netbox#12445] [Help Wanted!] [answered]
I don't use config contexts in Netbox. When I tried to create for the first time, i get this error:

`<class 'django.db.utils.IntegrityError'>`
`null value in column "data_path" of relation "extras_configcontext" violates not-null constraint
DETAIL:  Failing row contains (2023-05-03 09:38:27.972535+00, 2023-05-03 09:38:27.972558+00, 11, Test config context, 1000, , t, {"zabbix": {"proxy": "yourawesomeproxy.local"}}, null, null, null, null, null).`
`Python version: 3.10.6`
`NetBox version: 3.4.2`

Can you help me?

Thank you.

**Comment by candlerb** [comment]
Works for me (3.4.10), so the first step would be to upgrade to 3.4.10 in case this is an already-fixed bug.

If it still breaks with 3.4.10, then show a screenshot of exactly what you're entering to create the config context, and get a copy-pasteable backtrace (by setting DEBUG=True in configuration.py)

**Reply by candlerb** [reply]
On my system:

```
netbox=# \d extras_configcontext
                                Table "public.extras_configcontext"
    Column    |           Type           | Collation | Nullable |             Default
--------------+--------------------------+-----------+----------+----------------------------------
 created      | timestamp with time zone |           |          |
 last_updated | timestamp with time zone |           |          |
 id           | bigint                   |           | not null | generated by default as identity
 name         | character varying(100)   |           | not null |
 weight       | smallint                 |           | not null |
 description  | character varying(200)   |           | not null |
 is_active    | boolean                  |           | not null |
 data         | jsonb                    |           | not null |
```

No idea where that 'data_path' column came from.

**Reply by jeremystretch** [reply, marked as answer]
The `data_path` field was introduced in v3.5.0. You appear to be running v3.4.2 with a v3.5.0 database schema, somehow.

**Reply by gabrieldslacerda** [reply]
i see the problem. I thought I had restarted the services after updating to version 3.5, but didn't go through. 

Problem solved!
